### PR TITLE
Implement Info, cleanup, and post/delete notification V2 API tests

### LINKED ISSUE #444: [V2 notification] Implement info, POST and DELETE notification APIs
Implement the following APIs:
GET `/config`
GET `/version`
GET `/metrics`
GET `/ping`
DELETE `/cleanup`
DELETE `/cleanup/age/{age}`
POST `​/notification`
DELETE `/notification/id/{id}`
DELETE `/notification/age/{age}`

### PR DESCRIPTION
**Pull request by jinlinGuan**
fix #444

Signed-off-by: Ginny Guan <ginny@iotechsys.com>

### CODE REVIEW

**Comment by jinlinGuan** [comment]
The following testcase fails because of https://github.com/edgexfoundry/edgex-go/issues/3534
https://github.com/edgexfoundry/edgex-taf/blob/b57668a398475d4efdc0e62b8ccca8a7042bea02/TAF/testScenarios/functionalTest/V2-API/support-notifications/notification/DELETE.robot#L17-L27

File: TAF/testScenarios/functionalTest/V2-API/support-notifications/cleanup/DELETE.robot [hunk 1]
```diff
@@ -1,43 +1,87 @@
...
+    Given Create Subscriptions And Notifications
+    When Cleanup All Notifications And Transmissions By Age  ${age}
+    Then Should Return Status Code "202"
     And Should Return Content-Type "application/json"
     And Response Time Should Be Less Than "${default_response_time_threshold}"ms
     And All Notifications And Transmissions Under Age Should Not Be Found
+    [Teardown]  Run Keywords  Cleanup All Notifications And Transmissions By Age
+                ...      AND  Delete Subscription By Name ${subscription_names}[0]
 
 ErrCleanup001 - Delete transmission with invalid age
```
Comment (cherrycl) [inline TAF/testScenarios/functionalTest/V2-API/support-notifications/cleanup/DELETE.robot:38]:
```suggestion
ErrCleanup001 - Cleanup with invalid age
```

Comment (jinlinGuan) [reply TAF/testScenarios/functionalTest/V2-API/support-notifications/cleanup/DELETE.robot]: fixed

**Review by cherrycl — APPROVED** [review]
LGTM

**Pull request merged**